File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        mapping={}
        res=[]
        for i in range(0,len(nums)):
            need=target-nums[i]
            if need in mapping:
               
               res.append(mapping[need])
               res.append(i)
               
            else:
                mapping[nums[i]]=i
        return res

File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int n=nums.length;
        int []res= new int [n*2];
        for(int i=0;i<2*n;i++){
            res[i]=nums[i%n];
        }
        return res;
    }
}